lib/manageiq/rpm_build/generate_core.rb

```diff
@@ -43,6 +43,8 @@ def precompile_assets
       def precompile_sti_loader
         Dir.chdir(miq_dir) do
           shell_cmd("bundle exec rake evm:compile_sti_loader")
+
+          fixup_sti_loader!
```

Comment:
I do wonder if a `sed` script would work better for us.
It would be a little more blunt, but it may give us some flexibility going forward in our file format.

Reply:
Not sure why `sed` would be better than a proper YAML parser.

Reply:
Since we have to zero out the milliseconds sed is not an option (at least not easily or cleanly)